ci-operator/config/kata-containers/kata-containers/kata-containers-kata-containers-main.yaml

```diff
@@ -0,0 +1,100 @@
+base_images:
+  base:
+    name: "4.6"
```

Comment:
We are past 4.6.  Should this be bumped to 4.7?  Or does another config file need to exist to run on multiple versions?  Seems like we should be able to list the versions we want this job to run on.

Reply:
I tested this PR with 4.6, I prefer to merge it as is then send a another PR to bump the version.

Reply:
Perfect